doc/user_guide/athena_user_guide.md

```diff
@@ -0,0 +1,83 @@
+# Athena SQL Dialect User Guide
+
+The Athena SQL Dialect supports Amazon's [AWS Athena](https://aws.amazon.com/athena/), a managed service that lets you read files on S3 as if they were part of a relational database.
+
+## Registering the JDBC Driver in EXAOperation
+
+First download the [Athena JDBC driver](https://docs.aws.amazon.com/athena/latest/ug/connect-with-jdbc.html).
+
+Now register the driver in EXAOperation:
+
+1. Click "Software"
+1. Switch to tab "JDBC Drivers"
+1. Click "Browse..."
+1. Select JDBC driver file
+1. Click "Upload"
+1. Click "Add"
+1. In dialog "Add EXACluster JDBC driver" configure the JDBC driver (see below)
+
+You need to specify the following settings when adding the JDBC driver via EXAOperation.
+
+| Parameter | Value                                               |
+|-----------|-----------------------------------------------------|
+| Name      | `ATHENA`                                           |
+| Main      | `com.simba.athena.jdbc.Driver`                  |
+| Prefix    | `jdbc:awsathena:`                                 |
+| Files     | `AthenaJDBC42_<JDBC driver version>.jar`       |
```

Comment:
make it pretty please

Reply:
Yes, I wonder if the single quotes are messing this up. I change it locally but the remote version still shows this scrambled.